app/controllers/ProfilesController.php

```diff
@@ -88,7 +88,7 @@ public function createAction()
             ]);
 
             if (!$profile->save()) {
-                $this->flash->error($profile->getMessages());
+                $this->flash->error(join('<br/>', $profile->getMessages()));
```

Comment:
Why not use [nl2br](https://www.php.net/manual/en/function.nl2br.php)

Reply:
968b771